.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*                        @chamons @jhendrixMSFT
```

Comment:
Should this have a new-line?

Reply:
I used https://github.com/Azure/azure-sdk-for-go/blob/main/.github/CODEOWNERS#L9 as a template